apps/backend/src/board/logic/check-positions-is-connected.test.ts

```diff
@@ -0,0 +1,77 @@
+import { PathCard } from "~/models/card";
+import { Position } from "./get-available-positions";
+import {
+  getNeighbors,
+  isConnectNeighbor,
+} from "./check-positions-is-connected";
+import { Placement } from "~/models/placement";
+
+describe("check positions is connected", () => {
+  let board: Placement[];
+
+  beforeEach(() => {
+    board = [
+      { position: [0, 0], card: PathCard.START },
+      { position: [0, 1], card: PathCard.CONNECTED_BOTTOM_RIGHT },
+      { position: [1, 1], card: PathCard.CONNECTED_TOP_LEFT_RIGHT },
+      { position: [-1, 0], card: PathCard.DEADEND_TOP_BOTTOM_RIGHT },
+    ];
+  });
+
+  it.each<[Position, Position[]]>([
+    [
+      [1, 1],
+      [
+        [1, 2],
+        [1, 0],
+        [2, 1],
+        [0, 1],
+      ],
+    ],
+    [
+      [-1, -1],
+      [
+        [-1, 0],
+        [-1, -2],
+        [0, -1],
+        [-2, -1],
+      ],
+    ],
+  ])(
+    `
```

Comment:
Hello @nikkiching,
Could you help me update the test suit by

```
when:
  input position %j
expect:
  should return positions %j
```

This way maybe verbose, but it's much easier to check on CI log.